pkg/pillar/cmd/nim/nim.go

```diff
@@ -1030,7 +1030,7 @@ func (n *nim) makeLastResortDPC() (types.DevicePortConfig, error) {
 			},
 		}
 		dns := n.dpcManager.GetDNS()
-		portStatus := dns.GetPortByIfName(ifName)
+		portStatus := dns.LookupPortByIfName(ifName)
```

Comment:
this returned meaning should be 'portConfig', rather than status

Reply:
But this is from DNS, so there is status information additionally to the config.